vue2-vuetify/src/complex/OneOfTabRenderer.vue

```diff
@@ -0,0 +1,188 @@
+<template>
+  <div v-if="control.visible">
+    <combinator-properties
+      :schema="control.schema"
+      combinatorKeyword="oneOf"
+      :path="path"
+    />
+
+    <v-tabs v-model="tabIndex">
+      <v-tab
+        @change="handleTabChange"
+        v-for="(oneOfRenderInfo, oneOfIndex) in oneOfRenderInfos"
+        :key="`${control.path}-${oneOfIndex}`"
+      >
+        {{ oneOfRenderInfo.label }}
+      </v-tab>
+    </v-tabs>
+
+    <v-tabs-items v-model="selectedIndex">
+      <v-tab-item
+        v-for="(oneOfRenderInfo, oneOfIndex) in oneOfRenderInfos"
+        :key="`${control.path}-${oneOfIndex}`"
+      >
+        <dispatch-renderer
+          v-if="selectedIndex === oneOfIndex"
+          :schema="oneOfRenderInfo.schema"
+          :uischema="oneOfRenderInfo.uischema"
+          :path="control.path"
+          :renderers="control.renderers"
+          :cells="control.cells"
+          :enabled="control.enabled"
+        />
+      </v-tab-item>
+    </v-tabs-items>
+
+    <v-dialog v-model="dialog" persistent max-width="600" @keydown.esc="cancel">
+      <v-card>
+        <v-card-title class="text-h5"> Clear form? </v-card-title>
+
+        <v-card-text>
+          Your data will be cleared if you navigate away from this tab. Do you
+          want to proceed?
+        </v-card-text>
+
+        <v-card-actions>
+          <v-spacer></v-spacer>
+
+          <v-btn text @click="cancel"> No </v-btn>
+          <v-btn text ref="confirm" @click="confirm"> Yes </v-btn>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </div>
+</template>
+
+<script lang="ts">
+import {
+  and,
+  CombinatorSubSchemaRenderInfo,
+  ControlElement,
+  createCombinatorRenderInfos,
+  createDefaultValue,
+  isOneOfControl,
+  JsonFormsRendererRegistryEntry,
+  optionIs,
+  rankWith,
+} from '@jsonforms/core';
+import {
+  DispatchRenderer,
+  rendererProps,
+  RendererProps,
+  useJsonFormsOneOfControl,
+} from '@jsonforms/vue2';
+import isEmpty from 'lodash/isEmpty';
+import { defineComponent, ref } from 'vue';
+import {
+  VBtn,
+  VCard,
+  VCardActions,
+  VCardText,
+  VCardTitle,
+  VDialog,
+  VSpacer,
+  VTab,
+  VTabItem,
+  VTabs,
+  VTabsItems,
+} from 'vuetify/lib';
+import { useVuetifyControl } from '../util';
+import { CombinatorProperties } from './components';
+
+const controlRenderer = defineComponent({
+  name: 'one-of-renderer',
+  components: {
+    DispatchRenderer,
+    CombinatorProperties,
+    VDialog,
+    VCard,
+    VCardTitle,
+    VCardText,
+    VCardActions,
+    VSpacer,
+    VBtn,
+    VTabs,
+    VTab,
+    VTabsItems,
+    VTabItem,
+  },
+  props: {
+    ...rendererProps<ControlElement>(),
+  },
+  setup(props: RendererProps<ControlElement>) {
+    const input = useJsonFormsOneOfControl(props);
+    const control = (input.control as any).value as typeof input.control;
```

Comment:
Why can you not use `input.control` directly?

Reply:
the input.control is actually a a wrapper and inside the javascript/typescript those wrappers are not automatically outboxed - e.g. in the setup for ref objects you do need to used v.value instead just v for example.